src/lib/hello/posts.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Hello.Posts do
+  import Ecto.Query
+  alias Hello.Repo
+
+  alias Hello.Posts.Tweet
+
+  @doc """
+  Gets all tweets.
+  """
+  @spec get_all_tweets :: Tweet.t() | {:error, String.t()}
+  def get_all_tweets do
+    Tweet
+    |> order_by([t], desc: t.inserted_at)
+    |> Repo.all()
+    |> Repo.preload(:user)
```

Comment:
```suggestion
    Tweet
    |> preload(:user)
    |> order_by([t], desc: t.inserted_at)
    |> Repo.all()
```
Ecto.Queryのpreloadを使った方がよいです